README.md

```diff
@@ -0,0 +1,28 @@
+LLVM test suite Repository
+----------------------------
+
+Please see the LLVM testing infrastructure guide:
+
+  https://llvm.org/docs/TestSuiteGuide.html
+
+for more information on the contents of this repository.
+
+## Introduction
+
+Intel staging area for LLVM test suite contribution. Home for oneAPI Data
+Parallel C++ compiler tests extending LLVM test suite.
+
+## License
+
+See [LICENSE.txt](LICENSE.TXT) for details.
+
+## Contributing
+
+See [CONTRIBUTING.md](CONTRIBUTING.md) for details.
+
+## Sub-projects Documentation
+
+* oneAPI Data Parallel C++ compiler - See
+  [DPC++ Documentation](https://intel.github.io/llvm-docs/)
```

Comment:
This is not subproject of llvm-test-suite project. You can refer as some related project.